code/__HELPERS/path.dm

```diff
@@ -395,16 +395,8 @@
 
 	var/actual_dir = get_dir(src, destination_turf)
 
-	/// These are generally cheaper than looping contents so they go first
-	switch(destination_turf.pathing_pass_method)
-		if(TURF_PATHING_PASS_DENSITY)
-			if(destination_turf.density)
-				return TRUE
-		if(TURF_PATHING_PASS_PROC)
-			if(!destination_turf.CanPathfindPass(ID, actual_dir, caller, no_id = no_id))
-				return TRUE
-		if(TURF_PATHING_PASS_NO)
-			return TRUE
```

Comment:
Comment seems to suggest this can make things cheaper. Why remove these? Do they not work?

Reply:
It's never set to a different value so it always defaults to the density check